framework/runtime.cc

```diff
@@ -69,7 +69,7 @@ Finalize()
   material_stack.clear();
   multigroup_xs_stack.clear();
   function_stack.clear();
- 
+
   CALI_MARK_END("Simulation");
```

Comment:
Should this be:
```
CALI_MARK_END(opensn::name.c_str());
```
so that is matches the `CALI_MARK_BEGIN`?

Reply:
Yes. I forgot to get that in my first round of fixes. I have a final commit to this PR that fixes that, updates the install instructions, and updates `configure_dependencies.py`.